README.md

```diff
@@ -224,6 +224,13 @@ err = channel.PublishMultiple(ctx, []*ably.Message{
 if err != nil {
         panic(err)
 }
+
+// A REST client can publish messages on behalf of a realtime client
+// by providing the connection key of the realtime client.
+err := channel.Publish(ctx, "temperature", "12.7", ably.PublishWithConnectionKey("108yZeOBAB7iva!QQJ2CnCSa6A4ql22-a7443108yZeOBAB7iva"))
```

Comment:
I am not sure if we usually put secrets in the README, just a suggestion - 
`PublishWithConnectionKey("ConnectionKeyOfPublisherClientHere"));`

Reply:
This is just an example of realistic data, but I have no strong opinion and happy to make this change in order to get this PR through review. I've just pushed this change for you.

Reply:
Thanks !!